src/test/client.rs

```diff
@@ -500,3 +507,20 @@ async fn saslprep() {
     auth_test_uri("%E2%85%A8", "IV", None, true).await;
     auth_test_uri("%E2%85%A8", "I%C2%ADV", None, true).await;
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn watch_test() {
+    let client = TestClient::new().await;
+    if client.is_standalone() {
+        return;
+    }
+    let pipeline = vec![doc! { "$match": { "x": 3 }}];
+    let options = ChangeStreamOptions::builder()
+        .selection_criteria(SelectionCriteria::ReadPreference(ReadPreference::Primary))
+        .max_await_time(Duration::from_millis(5))
+        .batch_size(10)
+        .full_document(FullDocumentType::UpdateLookup)
+        .build();
+    let result = client.watch(pipeline, Some(options)).await.unwrap();
```

Comment:
I think this will cause a lint error due to `result` being unused. Since we don't actually need the change stream for this test until later, we can just skip binding it

Reply:
fixed